### DISCUSSION TITLE
QR code on bankwire.html for scan and pay

### DISCUSSION THREAD

**Discussion by malcek** [PrestaShop/PrestaShop#35024] [Ideas]
### The problem

Why there isn't already made QR code from data inside bankwire module and including order data It should work with SEPA UPN form for paymet in EU.

### The solution

Inside bankwire modul you set up everything that needs to be for payment over bank account.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by Hlavtox** [comment]
You can purchase dozens bankwire modules with QR code functionality.

Given that QR codes are local specific, it wouldn't be in the scope of the opensource project to maintain them anyway.
